server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
@matteofigus given this is in a 3.0 branch - and thus won't be deployed for a while - are you opposed to merging this in?